### ISSUE TITLE
Amd 470 vs 480 vs gtx 970

### ISSUE THREAD

**Issue by undefy** [nicehash/NiceHashMiner-Archived#568]
Hi all! Wish do you think is better?  For 470 and 480 is it better to get the 4gb or the 8gb model? Please comment your choice. 

470 has the best roi but 970 works like a charm on all algorithms...

**Comment by nerdatwork** [comment]
You can check here https://www.nicehash.com/index.jsp?p=calc and depending on your pocket you can choose which one you buy. Trust me I too was in your situation and nobody helped me out. I had to do everything on my own.

**Comment by undefy** [comment]
I was wondering about people's personal experience with them. I have all 3 and AMD is a bit of a pain in the ass BUT it is cheap and low power consumption. 6 cards of one power supply! Gtx is easy to set up and more versatile, higher re sale price,  but more expensive in the long run in terms of power consumption... 

What did you choose and why?

**Comment by nerdatwork** [comment]
AMD since lower price, faster ROI. Cryptocurrency prices go up and down so HIT when the iron is HOT I say :) I choose 480 always.

**Comment by snowcrawl** [comment]
My personal favorite in terms of price/power consumption is  the GTX 1060 (3GB version).  I try to pick them up when there is a sale.   I find that by comparison my 480 runs very hot, while the Nvidia card is just warm.   480 has been very solid and dependable though.

**Comment by undefy** [comment]
Well that is a new contestant! It is very strange.... It has the about the same ROI as a 480 or 470 however the specs are very different! e.g. 1060 has more processing power and ROP but less bus width, TMU and shaders. 

Please correct me if i am wrong but i thought the hashing rate was determined by processing power and bus width? So if it 1060 had more bus width it would be a super miner? Or did i get the basics wrong...

**Comment by snowcrawl** [comment]
At the moment the 1060 seems to be more versatile in terms of switching to different algorithms (7 different ones) based on profitability, whereas the 480 seems to be sticking to Equihash most of the time.  It seems as though some of these AMD cards have had some of the available miners taken away, like my   R9 380 which now only has 3 different algorithms listed in the benchmark section.  Until they get the Ethereum issue resolved for the AMD cards, it will probably just run Zcash all the time.

**Comment by undefy** [comment]
Which brand 1060 are you using? Asus?

**Comment by snowcrawl** [comment]
ZOTAC GeForce® GTX 1060 Mini.   I do have one that is the 6GB version, and it has a slight edge on certain algorithms, but overall probably not worth paying the extra money for the 6GB version.

**Comment by semisane** [comment]
I have 3 PC's running 8 different cards. 

One PC has a Asus Strix 1080 and a 980Ti, the second PC has a 980 and 970. The third has 4 x 480's. The AMD setup has been the biggest pain in my ass getting it to be stable. I love NHM for the nVidia PC's but between instability, driver issues, and NHM not playing nicely with the AMD cards my vote is for nVidia for long term investment. 
My nVidia machines never need to be restarted; AMD needs to be restarted/freezes/reboots at least twice a day.

**Comment by snowcrawl** [comment]
semisane, I think that if you keep working with configuration and perhaps with the help of the next releases of NHM, your AMD rig should get more stable.   I had to remove my weakest AMD card after this latest release.   An HD 7700 card, not important at all, able to only do less than a third of a regular mining card.  After I did that, and disabled ethereum mining for AMD,  it has been working flawlessly.

**Comment by semisane** [comment]
I've had the most problems with the addition of sgminer. The speeds reported by the miner were great but they were about half on NHM website. I went back to NHM 1.7.4.1 and have had much better luck with genoil. I do agree that tweaking the system should increase my stability. I've overclocked slightly and have had some success but have noticed that even with no OC'ing on my 4x 480's there are still sometimes reboots/freezes. 
I'd also like an update as to why the new AMD drivers don't work.
